client/src/common/diagnostic.ts

```diff
@@ -907,6 +907,34 @@ class DiagnosticFeatureProviderImpl implements DiagnosticProviderShape {
 			}
 		}));
 
+		tabs.onOpen((opened) => {
```

Comment:
This looks like it might return a disposable. Should that be handled?

Reply:
Great catch. Added to the list